src/constructs/core/stack.test.ts

```diff
@@ -10,17 +10,28 @@ import { TrackingTag } from "../../constants/library-info";
 import { GuStack } from "./stack";
 
 describe("The GuStack construct", () => {
-  it("should have stack and stage parameters", () => {
-    const stack = simpleGuStackForTesting();
+  it("requires passing the stack value as props", function () {
+    const stack = simpleGuStackForTesting({ stack: "some-stack" });
+    expect(stack.stack).toEqual("some-stack");
 
     const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
 
     expect(json.Parameters).toEqual({
```

Comment:
I don't think we need this assertion here, as it's tested as part of `"should have a stage parameter"`